src/OrchardCore/OrchardCore.ContentManagement/DefaultContentManager.cs

```diff
@@ -65,13 +65,13 @@ public async Task<ContentItem> NewAsync(string contentType)
             {
                 ContentType = contentTypeDefinition.Name,
                 Definition = contentTypeDefinition,
-                Builder = new ContentItemBuilder(contentTypeDefinition)
+                ContentItem = new ContentItem() { ContentType = contentTypeDefinition.Name }
```

Comment:
I think given the fix to `Weld` above that maybe there is no need to remove the `Builder` and change the API?

Reply:
We don't really need to but I thought the builder was a bit redundant and did not provide any value. Small cleanup while I was there.